servers/tejat-prior/default.nix

```diff
@@ -23,18 +23,27 @@ with lib;
     isSystemUser = true;
     useDefaultShell = true;
     group = "deploy";
-    openssh.authorizedKeys.keys = [ "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAINBuEKUhfJWZXUqgE2hN+aekbRj5yU8Q0kT4FjducocP webide" ];
+    openssh.authorizedKeys.keys = [
+      "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAINBuEKUhfJWZXUqgE2hN+aekbRj5yU8Q0kT4FjducocP webide"
```

Comment:
In general, I think it's a bad idea to use a single user for deploying multiple service. However, in this particular case, I think it's :good-enough: since we're planning to decommission our webide instance